test/logging.go

```diff
@@ -85,7 +86,12 @@ func newLogger(logLevel string) *zap.SugaredLogger {
 	  }
 	}`
 	configJSON := fmt.Sprintf(configJSONTemplate, logLevel)
-	l, _ := logging.NewLogger(string(configJSON), logLevel)
+	l, _, err := logging.NewLogger(string(configJSON), logLevel)
+	if err != nil {
+		l.Error("Failed to parse the logging config. Falling back to default logger.", zap.Error(err))
+	} else {
+		l.Infof("Successfully created the logger at level %s with config %s ", zap.String(logkey.JSONConfig, configJSON))
```

Comment:
i think in the test context we'd be okay with logging nothing in the success case here but @adrcunha can confirm